copy-resources.sh

```diff
@@ -10,5 +9,7 @@ mkdir -p ./src/test/resources/schemas/
 rm -rf ./src/test/resources/schemas/*
 rsync -avm --include='*/' --include='*' ./test/fixtures/ ./src/test/resources/schemas/
 # Build UI and copy files
-ng build
-node copy-bundled-ui.js || { echo "Failed to execute copy-bundled-ui.js"; }
+cd ui
+ng build || { echo "UI build failed"; exit 1; }
```

Comment:
I'm guessing you also need to run `npm ci` here too, as the `ui` folder also has a `package.json` file.